client/devdocs/design/index.jsx

```diff
@@ -83,44 +97,44 @@ export default React.createClass( {
 					</SearchCard>
 				}
 				<Collection component={ this.props.component } filter={ this.state.filter }>
-					<Accordions />
```

Comment:
Is there a chance to somehow automatically map the components to properties of `componentsUsageStats`?